revive/respawn.sqf

```diff
@@ -37,10 +37,13 @@ _unit setDamage 0;
 _unit setVariable ["compromised",0];
 _nul = [0,-1,getPos _unit] remoteExec ["citySupportChange",2];
 
-_hr = round ((server getVariable "hr") * 0.1);
-_resourcesFIA = round ((server getVariable "resourcesFIA") * 0.05);
-
-[- _hr, - _resourcesFIA] remoteExec ["resourcesFIA",2];
+if (isMultiplayer) then {

```

Comment:
write a comment that actually desctibes what this does, hard to read. maybe extract magic numbers here into local variables at least

```
private _hr_decrease_factor = ;
```